airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -22,6 +61,108 @@ create table "public"."airbyte_configs_migrations"(
+  "created_at" timestamptz(35) not null,
+  "updated_at" timestamptz(35) not null,
+  constraint "connection_pkey"
+    primary key ("id")
+);
+create table "public"."operation"(
+  "id" uuid not null,
+  "workspace_id" uuid not null,
+  "name" varchar(256) not null,
+  "operator_type" operator_type not null,
+  "operator_normalization" jsonb null,
+  "operator_dbt" jsonb null,
+  "tombstone" bool null,
+  "created_at" timestamptz(35) not null,
+  "updated_at" timestamptz(35) not null,
+  constraint "operation_pkey"
+    primary key ("id")
+);
+create table "public"."state"(
+  "id" uuid not null,
+  "connection_id" uuid not null,
+  "state" jsonb null,
+  "created_at" timestamptz(35) not null,
+  "updated_at" timestamptz(35) not null,
+  constraint "state_pkey"
+    primary key (
+      "id", 
+      "connection_id"
+    )
+);
+create table "public"."workspace"(
+  "id" uuid not null,
+  "customer_id" uuid null,
+  "name" varchar(256) not null,
+  "slug" varchar(256) not null,
+  "email" varchar(256) null,
+  "initial_setup_complete" bool not null,
+  "anonymous_data_collection" bool null,
+  "news" bool null,
+  "security_updates" bool null,
+  "display_setup_wizard" bool null,
+  "tombstone" bool null,
+  "notifications" jsonb null,
+  "first_completed_sync" bool null,
+  "feedback_done" bool null,
+  "created_at" timestamptz(35) not null,
+  "updated_at" timestamptz(35) not null,
+  constraint "workspace_pkey"
+    primary key ("id")
+);
+alter table "public"."actor"
+  add constraint "actor_actor_definition_id_fkey"
+    foreign key ("actor_definition_id")
+    references "public"."actor_definition" ("id");
+alter table "public"."actor"
+  add constraint "actor_workspace_id_fkey"
+    foreign key ("workspace_id")
+    references "public"."workspace" ("id");
+alter table "public"."actor_oauth_parameter"
+  add constraint "actor_oauth_parameter_actor_definition_id_fkey"
+    foreign key ("actor_definition_id")
+    references "public"."actor_definition" ("id");
+alter table "public"."actor_oauth_parameter"
+  add constraint "actor_oauth_parameter_workspace_id_fkey"
+    foreign key ("workspace_id")
+    references "public"."workspace" ("id");
+alter table "public"."connection"
+  add constraint "connection_destination_id_fkey"
+    foreign key ("destination_id")
+    references "public"."actor" ("id");
+alter table "public"."connection"
+  add constraint "connection_source_id_fkey"
+    foreign key ("source_id")
+    references "public"."actor" ("id");
+alter table "public"."operation"
+  add constraint "operation_workspace_id_fkey"
+    foreign key ("workspace_id")
+    references "public"."workspace" ("id");
+alter table "public"."state"
+  add constraint "state_connection_id_fkey"
+    foreign key ("connection_id")
+    references "public"."connection" ("id");
+create index "actor_actor_definition_id_idx" on "public"."actor"("actor_definition_id" asc);
+create index "actor_actor_type_idx" on "public"."actor"("actor_type" asc);
```

Comment:
i'm not sure how to think about this index. it's just going to bucket into one of two categories? what query is this making more efficient?

Reply:
same for the other two actor_type indexes. i don't think they will usually get used. usually for really low cardinality fields it doesn't make sense to do an index. this [SO article](https://stackoverflow.com/a/42972924) does an okay job explaining why.